README.md

```diff
@@ -347,6 +347,14 @@ foreman-maintain console
 foreman-maintain config
 ```
 
+## How to contribute?
+
+Generally, follow the [Foreman guidelines](https://theforeman.org/contribute.html). For code-related contributions, fork this project and send a pull request with all changes. Some things to keep in mind:
+* [Follow the rules](https://theforeman.org/contribute.html#SubmitPatches) about commit message style and create a [Redmine issue](http://projects.theforeman.org/projects/foreman-maintain/issues/new). Doing this right will help reviewers to get your contribution merged faster.
+* We have a [development handbook](https://theforeman.org/handbook.html) to help developers understand how Foreman developers code.
+* All of our pull requests run the full test suite in our [Jenkins CI system](http://ci.theforeman.org/). Please include tests in your pull requests for any additions or changes in functionality
```

Comment:
We don't use Jenkins for foreman-maintain as of now. We use Travis CI https://travis-ci.org/theforeman/foreman_maintain.
Also, please specify in commit message about the change. 'Update README' is too generic.

Otherwise looks good.